app/views/attendances/_attendance_table.html.erb

```diff
@@ -1,47 +1,31 @@
 <% content_for :head do %>
   <%= javascript_include_tag 'attendance_table' %>
 <% end %>
-
-<div class="container">
-<table id="attendance_table" class="table table-striped table-bordered" style="width:100%">
+<div class="container" style="overflow-x:auto;">
+   <% if @attendances_list.any? %>
+<table id="attendanceslist_table" style="width:100%" class="table data table-hover dt-responsive">
   <thead>
     <tr>
-      <th>Employee id</th>
-      <th>Name</th>
-      <th>Login Time</th>
-      <th>Logout Time</th>
+      <th >ID</th>
+      <th >NAME</th>
+      <th >EMAIL</th>
+      <th >LOGIN TIME</th>
+      <th >LOGOUT TIME</th>
     </tr>
   </thead>
-
   <tbody>
-    <% if @attendances_list.any? %>
     <% @attendances_list.each do |attendance| %>
-    <% @company_employee = Employee.find(attendance.employee_id) %>
+        <% @company_employee = Employee.find(attendance.employee_id) %>
```

Comment:
find_by

Reply:
resolved
changed to 
`attendance.employee`